docs/antrea-network-policy.md

```diff
@@ -255,8 +255,14 @@ default tier i.e. the "application" Tier.
 **ingress**: Each ClusterNetworkPolicy may consist of zero or more ordered
 set of ingress rules. Each rule, depending on the `action` field of the rule,
 allows or drops traffic which matches both the `from` and `ports` sections.
-Also, each rule has an optional `name` field, which is unique within the policy 
-describing the intention of this rule.
+Also, each rule has an optional `name` field, which should be unique within 
+the policy describing the intention of this rule. If users didn't provide `name`,
+antrea will auto-generate it. The auto-generate name will be in format: 
+ingress-[action]-[uid], e.g., 
+ingress-allow-2f0ed6ec1949367d0107ea931043df9eb430b1c3.
+The uid will be hash value of the rule based on sha1 algorithm. If a policy 
```

Comment:
```suggestion
the policy describing the intention of this rule. If `name` is not provided for 
a rule, it will be auto-generated by Antrea. The auto-generated name will be 
of format `ingress-[action]-[uid]`, e.g.
ingress-allow-2f0ed6ec1949367d0107ea931043df9eb430b1c3,
where [uid] is the hash value of the rule based on sha1 algorithm. If a policy 
```